src/SprykerEco/Zed/UnzerGui/Communication/Controller/DeleteMerchantUnzerCredentialsController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Communication\Controller;
+
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \SprykerEco\Zed\UnzerGui\Communication\UnzerGuiCommunicationFactory getFactory()
+ */
+class DeleteMerchantUnzerCredentialsController extends AbstractMerchantUnzerCredentialsController
+{
+    /**
+     * @var string
+     */
+    public const URL_MERCHANT_UNZER_CREDENTIALS_DELETE = '/unzer-gui/delete-merchant-unzer-credentials';
```

Comment:
```suggestion
    public const ROUTE_MERCHANT_UNZER_CREDENTIALS_DELETE = '/unzer-gui/delete-merchant-unzer-credentials';
```

ideally to have same naming pattern everywhere